docs/core/porting/versioning-sdk-msbuild-vs.md

```diff
@@ -7,43 +7,39 @@ ms.date: 06/10/2021
-| 3.1.4xx          | 16.7               | 16.7                                | Aug '20      | Dec '22   |
-| 5.0.1xx          | 16.8               | 16.8                                | November '20 | Mar '21   |
-| 5.0.2xx          | 16.9               | 16.8                                | March '21    | Aug '22   |
-| 5.0.3xx          | 16.10              | 16.8                                | May '21      | Aug '21   |
-| 5.0.4xx          | 16.11              | 16.8                                | Aug '21      | Feb '22*  |
-| 6.0.100-preview4 | 16.10 (No VS)      | 16.10-preview3                      | May '21      | N/A       |
-| 6.0.100-preview5 | 17.0-preview 1     | 16.10                               | June '21     | N/A       |
-| 6.0.100-preview6 | 17.0-preview 2     | 16.10                               | July         | N/A       |
-| 6.0.100-preview7 | 17.0-preview 3     | 16.10                               | August       | N/A       |
-| 6.0.100-rc1      | 17.0-preview 4     | 16.10                               | September    | N/A       |
-| 6.0.100          | 17.0               | 16.10**                             | Nov. '21     |           |
+| SDK Version      | MSBuild/VS version | Ship date    | Lifecycle |
+|------------------|--------------------|--------------|-----------|
+| 2.1.5xx          | 15.9               | Nov '18      | Aug '21*  |
+| 2.1.8xx          | 16.2 (No VS)       | July '19     | Aug '21   |
+| 3.1.1xx          | 16.4               | Dec '19      | Dec '22   |
+| 3.1.4xx          | 16.7               | Aug '20      | Dec '22   |
+| 5.0.1xx          | 16.8               | November '20 | Mar '21   |
+| 5.0.2xx          | 16.9               | March '21    | Aug '22   |
+| 5.0.3xx          | 16.10              | May '21      | Aug '21   |
+| 5.0.4xx          | 16.11              | Aug '21      | Feb '22*  |
+| 6.0.100-rc1      | 17.0-preview 4     | September    | N/A       |
+| 6.0.100          | 17.0**             | Nov. '21     |           |
 
 > [!NOTE]
-> The Minimum version is what's required to load the SDK but doesn't cover what's supported. Targeting net6.0 is officially supported in 17.0+ only.
+> Targeting net6.0 is officially supported in 17.0+ only.
 
 > \* MSbuild/VS supported for longer
 >
-> \*\* You won't have to upgrade to 17.0 on the day .NET 6 releases. This is the current version and will likely be 16.11 at release. This version supports downlevel targeting, such as .NET 5 and below, but doesn't allow targeting .NET 6 for the purpose of using .NET 6 or C# 10 features.
+> \*\* The goal is on .NET 6 release day is for the SDK to be functional in 16.11 for **downlevel** targeting so customers are not forced to update their SDK and VS versions simultaneously. However, this will NOT allow for targeting .NET 6 because of limitations in 6.0 features and C#10 features in 16.11. This is specifically for targeting 5.0 and below.
```

Comment:
```suggestion
> \*\* When .NET 6 releases, the goal is for the .NET SDK to be functional in version 16.11 for **downlevel** targeting. This means that you're not forced to update your SDK and Visual Studio versions simultaneously. However, you won't be able to target .NET 6 because of limitations in 6.0 features and C#10 features in version 16.11. This compatibility is specifically for targeting 5.0 and below.
```